### PR TITLE
Adds integrity check (#19)

### PR DESCRIPTION
**Pull request by RaMin0**

### CODE REVIEW

**Comment by paked** [comment]
@RaMin0: Thanks for submitting this PR :)

I'm pretty busy right now, but hopefully I can get around to it in the next 48 hours.

If I don't, feel free to pester me.

**Comment by RaMin0** [comment]
@paked Dong Dong Dong 🛎

File: cmd/bot/main.go [hunk 1]
```diff
@@ -10,13 +11,18 @@ import (
 )
 
 var (
+	port = flag.Int("p", 8080, "The port used to serve the messenger bot")
```
Comment (paked) [inline cmd/bot/main.go]: I'd prefer to only use one configuration package.

Could you rewrite this as:

`port = Flag.Int("port", 8080, "The port used to serve the messenger bot")`

You can provide the parameter as `./messenger-bot --port=4040` in the same way you would a normal `flag`.

Remember to remove all references to the `flag` package.

**Review by paked — CHANGES_REQUESTED** [review]
Thanks for submitting this PR! I've been meaning to write this code for quite a while now.

One change, and then we're good to go. Thank you :heart:

File: cmd/bot/main.go (hunk 1 above)
Comment (RaMin0) [reply cmd/bot/main.go]: @paked Fixed in c3bdb1b.

**Comment by paked** [comment]
Thank you!

**Pull request merged**